core/keyserver/epochs_test.go

```diff
@@ -70,6 +70,15 @@ func TestListMutations(t *testing.T) {
 		t.Fatalf("admin.Write(): %v", err)
 	}
 
+	// Log server
+	numSequencers := 1
+	unused := ""
+	logEnv, err := integration.NewLogEnv(ctx, numSequencers, unused)
```

Comment:
optionally:
`logEnv, err := integration.NewLogEnv(ctx, 1 /* numSequencers */ , "" /* unused */)`

Reply:
not sure if this is better.